systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java

```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
     }
 
     public WebElement getEditMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Edit')]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='edit-address']/div"));
     }
 
     public WebElement getDeleteMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Delete')]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='delete-address']/div"));
     }
 
     public WebElement getPurgeMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Purge')])]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='purge-address']/div"));
```

Comment:
if you use id in element, use .findElement(By.id("some_id"))

Reply:
@kornys Okay. But in this case, the element is a div and doesn't contain id, so I think xpath is required. 
Here is the element -
<li role="menuitem" id="edit-address">
   <div tabindex="-1" data-key="0" class="pf-c-dropdown__menu-item">Edit</div>
</li>

Reply:
You can select li element which has id, whatever element you need.